src/components/Meal/Meal.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
+import data from '@groceristar/groceristar-fetch/chickenKyiv';
+import DisplayList from '../DisplayList/DisplayList'
+
+class Meal extends Component {
```

Comment:
Got it. Sorry, i think i was not clear. My idea was to have a router, that will display data, related to grocery lists.
like if you open /xxx/:id - you'll see a single grocery list with all necessary data, as we have at tiles.

But, as we added meal components and route - let's keep it - we'll use it somehow later.